server/model/gameSessionManager.ts

```diff
@@ -1,8 +1,8 @@
 import Player from "./actors/player";
 
 export class gameSessionManager {
-  private player1: Player;
-  private player2: Player;
+  public player1: Player;
+  public player2: Player;
   private static rules: Record<string, string> = {
     rock: "scissors",
```

Comment:
These "rules" (actions) should probably be moved into a separate class and maybe changed into a different type, like an Enum (this might just be personal preference thou)